packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -656,13 +671,25 @@ class _CheckboxPainter extends ToggleablePainter {
     notifyListeners();
   }
 
-  BorderSide? get side => _side;
-  BorderSide? _side;
-  set side(BorderSide? value) {
-    if (_side == value) {
+  BorderSide get activeSide => _activeSide!;
+  BorderSide? _activeSide;
+  set activeSide(BorderSide? value) {
+    assert(value != null);
```

Comment:
So `set activeSide(BorderSide value) {` instead of this assert And the getter wouldn't need a `!`.

Reply:
Got it. Fixing!